src/twisted/test/test_task.py

```diff
@@ -1205,3 +1205,214 @@ def main(reactor):
         r = _FakeReactor()
         exitError = self.assertRaises(SystemExit, task.react, main, [], _reactor=r)
         self.assertEqual(0, exitError.code)
+
+
+class ReactCoroFnTests(unittest.SynchronousTestCase):
```

Comment:
Readability matters, so maybe rename `ReactCoroFnTests` to `ReactCoroutineFunctionTests`. Length here is harmless.

Reply:
```suggestion
class ReactCoroutineFunctionTests(unittest.SynchronousTestCase):
```